src/View/TwigView.php

```diff
@@ -26,7 +26,7 @@
  * Class TwigView.
  * @package WyriHaximus\TwigView\View
  */
-class TwigView extends View
+class TwigView extends View // FIXME: PHP Fatal error:  Class App\View\AppView may not inherit from final class (WyriHaximus\TwigView\View\TwigView)
```

Comment:
Now I don't understand why App/View/AppView inherits from TwigView but that seems the way CakePHP internally works. Probably not good but nothing we can change unless it changes for CakePHP4?
Question: Is this the immutability by using composition/decoration instead of inheritance that gives benefits?

Reply:
We can't make the `TwigView` `final` because of the extension sadly enough. The only way would be CakePHP4 changing this behavior.

Well immutability  is something different then composition/decoration. PSR-7 for example has immutability. Where composition is about having small classes that do one thing well so others can compose their behavior using those small classes/

Reply:
I don't mean immutability of data but of of schemas/structures (classes). It tries to inherit and change some methods (mutate the method declaration). Anyway I get it (I hope)

Reply:
Ah right nvm then. It overwrites the rendering within the default view. But anyway can you remove the `FIXME`? Not going to happen in the near future that we don't extend it.